### DISCUSSION TITLE
Surface tagging - difference between "ground" and "unpaved" is unclear

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#3440] [Feedback]
Users reported being confused by what is the difference between "ground" and "unpaved".

I admit that it also was not really clear to me.

I have no good idea how to fix it.

People in general are often confused by surface quest, some were unsure for example about difference between `concrete` `asphalt` and `paving_stones`

Multiple independently requested access to more thorough info about surface tagging.

It is not helping that OSM Wiki also has quite poor documentation (say, what is actually relation between `surface=dirt` and `surface=earth`? OSM Wiki still is not making clear that it is a de facto duplicate)

![screen](https://user-images.githubusercontent.com/899988/138585900-ebc71f6e-d1cc-45a3-b446-4a66751b5571.png)

**Comment by smichel17** [comment]
#1913 seems like it would help

On October 24, 2021 8:14:23 AM UTC, Mateusz Konieczny ***@***.***> wrote:
>Users reported being confused by what is the difference between "ground" and "unpaved".
>
>I admit that it also was not really clear to me.
>
>I have no good idea how to fix it.
>
>People in general are often confused by surface quest, some were unsure for example about difference between `concrete` `asphalt` and `paving_stones`
>
>Multiple independently requested access to more thorough info about surface tagging.
>
>It is not helping that OSM Wiki also has quite poor documentation (say, what is actually relation between `surface=dirt` and `surface=earth`? OSM Wiki still is not making clear that it is a de facto duplicate)
>
>![screen](https://user-images.githubusercontent.com/899988/138585900-ebc71f6e-d1cc-45a3-b446-4a66751b5571.png)
>
>

-- 
To respect your time, I try to write short, functional emails.

**Comment by westnordost** [comment]
> I admit that it also was not really clear to me.

Was, or is? 

Ground is a natural ground surface. E.g. a trail.

Unpaved is a man-made surface that is not sealed. (Not sure if it is Germanenglish to say that. In German, "sealed" surfaces are surfaces like asphalt, concrete, paving stones etc.)

**Reply by westnordost** [reply]
So, maybe "natural ground" would work? Maybe "unsealed" would work, if it is understood?

**Reply by andrewharvey** [reply]
> Unpaved is a man-made surface that is not sealed. (Not sure if it is Germanenglish to say that. In German, "sealed" surfaces are surfaces like asphalt, concrete, paving stones etc.)

That's not my understanding, and doesn't reflect how I've seen the tag widly used. `unpaved` is just anything that's not sealed, it could be dirt, gravel, sand, compacted. ie. not paved/sealed.

**Reply by mnalis** [reply]
"sealed" / "unsealed" does not really work for me in English as a replacement for "paved" / "unpaved" (which feels natural to me there)...

- `surface=ground` to me is generic natural ground (unmodified by humans by adding external materials, but often modified by mechanical action like walking or driving over it), but which is not of exactly one type (so it could be at one place mostly dirt, in some places with some rock/gravel, in some places more sandy or with little grass, mud etc).
- `unpaved` is even more generic - either plain `ground` above, or ground improved with human-added material which is in loose condition (compacted, gravel, ...)
- `paved` is generic for surface which is human-covered with material which is not loose (like paving stones, concrete, asphalt...)

So if one were to just build a binary tree classification, every surface would be either in `paved` or in `unpaved` branch (and then those in `unpaved` would branch to `compacted`, `gravel`, ... , `ground`; and then those in `ground` branch would further split to `mud`, `dirt`, `grass`, ...)

How to convey that in a word or two is beyond me, though.

As for the difference between `concrete` and `asphalt`, I do not get it why they get confused? Those are quite different materials, and while they have some similarities,  they also have differences which should be visible both when old and new, except maybe when it is very dark?

Some `paving_stones` might be made of concrete, which could cause the confusion (and with new `concrete_plates` option, it is probably not going to get better - those are essentially _very big_  paving stones _made of concrete_). Not sure how to explain it without at least a full sentence or two.

---

As for translated versions, only the natural speakers can tell what fits well in their language, I guess.

In Croatian it is a mess anyway, as we seem to completely lack generic words for paved/unpaved :smile:, so we resort to using either "neasfaltiran" (literally "not"+"covered by asphalt") or "nepopoločan" (literally "not" + "covered by paving stones") to convey the idea. While both are often used in generic sense to mean "unpaved", they also sometimes retain their _more specific_ meaning, so it is impossible to convey exact meaning without adding that `(generic)`.

**Reply by matkoniecz** [reply]
> >  I admit that it also was not really clear to me.

> Was, or is?

Only after going through OSM Wiki and discussing it in person I understood that `ground` is intended to be covering mix of various natural surfaces such as dirt/rocks/roots/leaves. And still I am not entirely sure is there benefit in using it over `unpaved`

**Reply by mnalis** [reply]
@matkoniecz well, `ground` is one on more popular surfaces I use when mapping outdoors out-of-city. Wilderness surface is often intermixed in small intervals here, and I mostly do not care enough to micromap to such levels (eg. _"5m dirt, then 1m grass, then 3m dirt, then 1m mud, then 3m grass..."_)

On the other hand, I rarely use `unpaved` - while I understood wiki and other people will use it even for `ground`, I personally would use it only if it is **not** `ground`, and I can't be bothered to split `compacted` / `gravel` -- situtation which happens **_very rarely_** (because, as a cyclist mostly on trekking bike, I care _**very much**_  for difference between `compacted` and `gravel`). 

But I guess that people with sport cars might find `unpaved` value useful as they don't care about details of it (_"meh, it's not asphalt nor concrete, but some worthress unpaved road which I'd never use, so I don't care if it is gravel or dirt or mud..."_)

**Reply by matkoniecz** [reply]
> Unpaved is a man-made surface that is not sealed. 

I am pretty sure that it is not how this tag is used and I am unaware about places where it is described in this way. Maybe German has separate words for natural-unpaved and man-made-unpaved?

**Reply by matkoniecz** [reply]
> As for the difference between concrete and asphalt, I do not get it why they get confused? 

I also was surprised.

They simply never though about classifying road surfaces, and were never considering it. In the same way as before related StreetComplete quest I never noticed vibrating buttons under traffic lights ( https://wiki.openstreetmap.org/wiki/Key:traffic_signals:vibration )

**Reply by westnordost** [reply]
When I was a child, there was a rumor that this hidden button would grant us green light quicker, so we pressed it all the time :-)
